bigquery/deploy/deploy-indexer.sh

```diff
@@ -44,3 +46,10 @@ kubectl create configmap dataset-config --from-file=../../dataset_config/${datas
 # it may not yet exist.
 kubectl delete -f bq-indexer.yaml || true
 kubectl create -f bq-indexer.yaml
+
+echo "Waiting for indexer to complete..."
+job_name=$(kubectl get pods | grep bq-indexer | awk '{print $1}')
+kubectl wait --for=condition=complete "job.batch/bq-indexer" --timeout=-1s
```

Comment:
I get this error:
```
Error from server (NotFound): configmaps "dataset-config" not found
configmap "dataset-config" created
Error from server (NotFound): error when stopping "bq-indexer.yaml": jobs.batch "bq-indexer" not found
job.batch "bq-indexer" created
Waiting for indexer to complete...
Error: unknown command "wait" for "kubectl"

Did you mean this?
	edit
	taint

Run 'kubectl --help' for usage.
unknown command "wait" for "kubectl"

Did you mean this?
	edit
	taint
```
Looks like `kubectl wait` was [introduced in 1.11](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.11.md).

I couldn't figure out how to upgrade kubectl; `gcloud components update kubectl --version 1.11` doesn't work.

Please either:
- At the top of the script, if kubectl client version < 1.11, exit with an error like "This script uses "kubectl wait" which was introduced in kubectl 1.11. Please upgrade with UPGRADE_COMMAND."
- Stop using `kubectl wait` and use bash for loop. Add TODO for `kubectl wait`.